corona-warn-app/cwa-app-android
# Fix loading detection tracking data (EXPOSUREAPP-14112)

Test
- Install 2.27.2
- Use test menu to have a high risk 
- upgrade to 2.28.0 (this branch) last time calculations should be displayed properly 


`startedAt` , `finishedAt` were saved before as objects  such
```json
"finishedAt": {
      "iMillis": 1603473968125
 }
```
reading this as Long will lead to parsing issue, to avoid this issue, old persisted data should be read as Object and then save timestamps as longs 
Ticket
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14112
